<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>1111</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Since.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>-14</y>
      <width>331</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>TEC-Galery (2).png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>247</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_User"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Password: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_Pass"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="SignIn">
       <property name="text">
        <string>Sign in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Register</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>If you don't have an account,</string>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>pliss click in &quot;Sign Up&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>line_Pass</zorder>
   <zorder>label</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
